File: Java labs/lab3/src/com/iot/model/Drink.java
package com.iot.model;

import model.enums.DrinkType;
import model.enums.DrinkSortType;

public class Drink {

	private String name = "Water";
	private int capacity = 250;
	private int price;
	private int milkAmount;
	private DrinkSortType sort = DrinkSortType.OTHER;
	private DrinkType drinkType = DrinkType.WATER;

	public Drink() {
	}

	public Drink(String name, int capacity, int price, DrinkSortType sort, int milkAmount, DrinkType drinkType) {
		super();
		this.name = name;
		this.capacity = capacity;
		this.price = price;
		this.sort = sort;
		this.milkAmount = milkAmount;
		this.drinkType = drinkType;
	}

	@Override
	public String toString() {
		String info = "other: " + getName() + ", ";
		info += getCapacity() + "ml, ";
		info += getPrice() + "$";
		if (getMilkAmount() > 0) {
			info += ", milk: " + getMilkAmount() + "ml ";
		}
		return info;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public int getCapacity() {
		return capacity;
	}

	public void setCapacity(int capacity) {
		this.capacity = capacity;
	}

	public int getPrice() {
		return price;
	}

	public void setPrice(int price) {
		this.price = price;
	}

	public int getMilkAmount() {
		return milkAmount;
	}

	public void setMilkAmount(int milkAmount) {
		this.milkAmount = milkAmount;
	}

	public DrinkType getDrinkType() {
		return drinkType;
	}

	public DrinkSortType getSort() {
		return sort;
	}

	public void setSort(DrinkSortType sort) {
		this.sort = sort;
	}

}

File: Python labs/lab2/Drink.py
from DrinkSortType import DrinkSortType
from DrinkType import DrinkType


class Drink:
    name, capacity, price, sort_of_drink, drink_type, coffee_amount = 'Water', 250, 0, DrinkSortType.OTHER, DrinkType.OTHER, 0

    def __init__(self, m_name="Water", m_capacity=250, m_price=0, m_sort=DrinkSortType.OTHER,
                 m_drink_type=DrinkType.OTHER):
        self.name = m_name
        self.capacity = m_capacity
        self.price = m_price
        self.sort_of_drink = m_sort
        self.drink_type = m_drink_type

    def reset_values(self, m_name, m_capacity, m_price, m_sort, m_drink_type):
        self.name = m_name
        self.capacity = m_capacity
        self.price = m_price
        self.sort_of_drink = m_sort
        self.drink_type = m_drink_type

    def to_string(self):
        return 'other: ' + self.name + ', ' + str(self.capacity) + 'ml, ' + str(self.price) + '$'

